src/main/resources/templates/start.html

```diff
@@ -26,35 +26,7 @@
     <title>Home</title>
 </head>
 <body onload="bindClickHandlers()">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="/">TvTracker</a>
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarSupportedContent">
-                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="/">Browse</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/favorites">Favorites</a>
-                    </li>
-                </ul>
-
-
-                <div class="d-flex">
-                    <a class="nav-link" href="#">Login</a>
-                    <select id="mediaTypeDropdown">
-                        <option value="movie">Movie</option>
-                        <option value="series">Series</option>
-                    </select>
-                    <input id="searchTextBox" class="form-control me-2" type="text" placeholder="Search" aria-label="Search">
-                    <button id="searchBtn" class="btn btn-outline-success">Search</button>
-                </div>
-            </div>
-        </div>
-    </nav>
+<div th:replace="fragments/navbar :: navbar"></div>
```

Comment:
I added the thymeleaf tag to load navbar.html into a div in this start.html page. Clean code also reduces technical debt as discussed here https://martinfowler.com/bliki/TechnicalDebt.html

Reply:
Good.